tyk-pro/templates/bootstrap-post-install.yaml

```diff
@@ -27,73 +27,62 @@ spec:
     spec:
       serviceAccountName: k8s-bootstrap-role
       containers:
-      - name: bootstrap-post-install
-        image: bash:5.1
-        imagePullPolicy: IfNotPresent
-        command: 
-        - sh
-        - -c
-        - /opt/scripts/init.sh
-        env:
-          - name: TYK_ADMIN_FIRST_NAME
-            value: {{ .Values.dash.adminUser.firstName | quote }}
-          - name: TYK_ADMIN_LAST_NAME
-            value: {{ .Values.dash.adminUser.lastName | quote }}
-          - name: TYK_ADMIN_EMAIL
-            value: {{ .Values.dash.adminUser.email | quote }}
-          - name: TYK_ADMIN_PASSWORD
-            valueFrom:
-              secretKeyRef:
-                name: {{ .Release.Name }}-login-details
-                key: TYK_PASS
-          - name: TYK_POD_NAMESPACE
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.namespace
-          - name: TYK_DASHBOARD_PROTO
-            value: {{ include "tyk-pro.dash_proto" . }}
-          - name: TYK_DASHBOARD_SVC
-            value: dashboard-svc-{{ include "tyk-pro.fullname" . }}
-          - name: TYK_DB_LISTENPORT
-            value: "{{ .Values.dash.containerPort }}"
-          - name: TYK_DB_LICENSEKEY
-            value: {{ .Values.dash.license | quote }}
-          - name: TYK_ADMIN_SECRET
-            value: {{ .Values.secrets.AdminSecret | quote }}
-          - name: TYK_ORG_NAME
-            value: {{ .Values.dash.org.name | quote }}
-          - name: TYK_ORG_CNAME
-            value: {{ .Values.dash.org.cname | quote }}
-          {{- if .Values.dash.enabled }}
-          - name: DASHBOARD_ENABLED
-            value: "true"
-          {{- end }}
-          {{- if .Values.dash.operatorSecret }}
-          - name: OPERATOR_SECRET_ENABLED
-            value: "true"
-          - name: OPERATOR_SECRET_NAME
-            value: "tyk-operator-conf"
-          {{- end }}
-          {{- if .Values.dash.bootstrap }}
-          - name: BOOTSTRAP_DASHBOARD
-            value: "true"
-          {{- end }}
-          - name: GATEWAY_ADDRESS
-            value: {{ include "tyk-pro.gateway_url" . }}
-          {{- if .Values.portal.bootstrap }}
-          - name: BOOTSTRAP_PORTAL
-            value: "true"
-          {{- end }}
-          - name: TYK_DASHBOARD_DEPLOY
-            value: dashboard-{{ include "tyk-pro.fullname" . }}
-        volumeMounts:
-          - name: bootstrap-config-volume
-            mountPath: /opt/scripts
-      volumes:
-        - name: bootstrap-config-volume
-          configMap:
-            name: bootstrap-post-install-configmap-{{ include "tyk-pro.fullname" . }}
-            defaultMode: 0777
+        - name: bootstrap-tyk
```

Comment:
WDYT updating it as `bootstrap-tyk-post-install`?

Reply:
Done.